File: src/Java/Nasty Hacks/NastyHacks.java
public class NastyHacks {
    public static void main(String args[]) {
        Kattio io = new Kattio(System.in);
        int n = io.getInt();
        for (int i = 0; i < n; i++) {
            int r = io.getInt(), eminusc = io.getInt() - io.getInt();
            if (r < eminusc) {
                io.println("advertise");
            } else if (r == eminusc) {
                io.println("does not matter");
            } else {
                io.println("do not advertise");
            }
            io.flush();
        }
    io.flush(); 
    }
}

File: src/Python 3/Nasty Hacks/NastyHacks.py
for i in range(int(input())):
    r, e, c = input().split()
    r = int(r)
    e = int(e)
    c = int(c)
    if r < e - c:
        print('advertise')
    elif r == e - c:
        print('does not matter')
    elif r > e - c:
        print('do not advertise')